.devcontainer/Dockerfile

```diff
@@ -12,28 +12,40 @@
 ################################################################################
 
 FROM mcr.microsoft.com/devcontainers/rust:latest
-
 ARG USERNAME=vscode
 ARG TARGETARCH
 
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get -y install \
-    # Others
     curl \
-    zip \
     gcc \
     git \
+    openjdk-17-jre \
+    zip \
     && \
     rm -rf /var/lib/apt/lists/*
 
+# Download openfasttrace JARs and create shortcut oft executable
+ARG OFT_CORE_VERSION=4.1.0
+ARG OFT_ASCIIDOC_PLUGIN_VERSION=0.3.0
+ENV LIB_DIR=/opt/oft/lib
+RUN <<EOF
+mkdir -p $LIB_DIR
+base_url=https://github.com/itsallcode
+wget -P $LIB_DIR ${base_url}/openfasttrace/releases/download/$OFT_CORE_VERSION/openfasttrace-$OFT_CORE_VERSION.jar
+wget -P $LIB_DIR ${base_url}/openfasttrace-asciidoc-plugin/releases/download/$OFT_ASCIIDOC_PLUGIN_VERSION/openfasttrace-asciidoc-plugin-$OFT_ASCIIDOC_PLUGIN_VERSION-with-dependencies.jar
+EOF
```

Comment:
I am not sure about the outcome if any of these individual commands fail. Concatenating them with `&&` might be a better choice to make sure that they either all succeed or fail.

Reply:
done